rn/Teacher/src/modules/submissions/list/get-submissions-props.js

```diff
@@ -104,34 +93,31 @@ function submissionProps (enrollment: Enrollment, submission: ?SubmissionWithHis
   return { userID: id, avatarURL, name, status, grade, submissionID, submission, score, sectionID, allSectionIDs }
 }
 
-export function dueDate (assignment: Assignment, user: ?User | SessionUser): ?string {
+export function dueDate (assignment: Assignment, user: ?User | SessionUser, group: ?Group): ?string {
   if (!assignment) {
-    return null
+    return
   }
 
   const overrides = assignment.overrides
-  if (overrides) {
-    const override = find(overrides, (override) => {
-      if (!override.student_ids) return false
+  if (!overrides) {
+    return assignment.due_at
+  }
+
+  let override
+  if (group) {
+    override = overrides.find(override => {
+      return override.group_id != null &&
+             group &&
```

Comment:
Despite checking this on line 107, flow couldn't reconcile that and so have to check here again.